index.js

```diff
@@ -288,6 +292,12 @@ BFFS.prototype.publish = function publish(spec, options, fn) {
     }
 
     if (error) return fn(error);
+
+    if (!error) {
```

Comment:
No need to wrap this in an `if`, since the prior line returns on error. You won't get to this line of code if `error` is `truthy`.